app/styles/sidebar.scss

```diff
@@ -0,0 +1,120 @@
+// sidebar
+.sidebar-logo {
+  font-size: 3.5rem;
+  margin-top: 15px;
+  margin-bottom: 75px;
+  text-align: center;
+  font-family: 'Shadows Into Light', cursive;
+}
+
+ul.side-nav.fixed li.logo {
```

Comment:
Good catch with using element names. Point of not using them is to avoid need of rewritng styles to apply them in another case (where same html structure does not have to be used).